src/components/ImageView/ImageView.js

```diff
@@ -454,6 +454,13 @@ export default observer(
     imageRef = createRef();
     crosshairRef = createRef();
 
+    constructor(props) {
+      super(props);
+
+      if(typeof props.item.smoothing === 'boolean')
```

Comment:
See my below comment about the optional with default instead of maybeNull, this could be then simplified to and no type check required:

```js
props.store.settings.setSmoothing(props.item.smoothing);
```